api/build.gradle

```diff
@@ -139,6 +142,7 @@ dependencies {
 
   testCompile 'junit:junit:4.12'
   testCompile 'org.mockito:mockito-core:1.+'
+  testCompile 'com.google.guava:guava:+' // required for com.google.truth:truth
```

Comment:
Hmm. I've been using truth without this so far without issue. What issue did you run into?